internal/scan.go

```diff
@@ -53,6 +53,7 @@ func Scan(cmd *cobra.Command, args []string) {
 	}
 	reporters = append(reporters, &reporting.ConsoleReporter{Config: userConfig})
 
+	reportTime := time.Now().Format(time.RFC1123)
```

Comment:
[q] do we want to explicitly set `UTC()` here?
Looking at the source code, I don't think it actually changes anything
https://cs.opensource.google/go/go/+/master:src/time/time.go;l=1127-1131

since `setLoc` ignores UTC
https://cs.opensource.google/go/go/+/refs/heads/master:src/time/time.go;l=205-212;drc=c5ba9d2232695816a3bd55d360aa6ef7aa1f63ed

Reply:
This is a great call! Especially when we create the Datadog reporter, I think we will definitely want to have this normalized to UTC. And in fact, taking it one step further, I think that we will want to pass just the raw timestamp to reporters and let them format it.

Even one step further beyond that, we can introduce a config option to let users control the formatting of the timestamp. 😄